src/actions.js

```diff
@@ -1,3 +1,55 @@
 import { createAction } from 'redux-actions';
+import SC from 'soundcloud';
+import _ from 'lodash';
 
 export const addFun = createAction('ADD_FUN');
+export const getTracksFromSC = createAction('GET_TRACKS_FROM_SC');
+
+// async
+const DEFAULT_USER_ID = '7742327';
+
+SC.initialize({
+  client_id: process.env.REACT_APP_SC_CLIENT_ID,
+  redirect_uri: process.env.REACT_APP_REDIRECT_URI
+});
+
+export const getTracks = () => {
+  return dispatch => {
```

Comment:
`getTracks()` is no longer testable on its own. You now need a whole redux dispatcher system set up to run that test.

Reply:
I think your goal should be a thunk action like this:

```js
export const getTracks = () {
  return dispatch => {
    scLibraryMethod
      .then(tracks => dispatch(getTracksFromSC(tracks))
      .catch(e => dispatch(getTracksFromSC(e));    	 
  }
}
```